src/screens/SearchScreens/SearchScreen.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import { Box, TextField, Typography } from "@mui/material";
+import { SearchOutlined } from "@mui/icons-material";
+import InputAdornment from "@mui/material/InputAdornment";
+import { Layout } from "@forkfacts/components";
+import { useStyles } from "./styles";
+
+export default function SearchScreen() {
```

Comment:
Wrong name. This is home page for forkfacts. The name should be `HomeScreen`.

Reply:
Rename the story and file names accordingly

Reply:
Fixed and renamed